generator/src/main/java/com/linkedin/pegasus/generator/PegasusDataTemplateGenerator.java

```diff
@@ -97,7 +98,15 @@ public static void main(String[] args)
 
     final String generateImportedProperty = System.getProperty(PegasusDataTemplateGenerator.GENERATOR_GENERATE_IMPORTED);
     final boolean generateImported = generateImportedProperty == null ? true : Boolean.parseBoolean(generateImportedProperty);
-    PegasusDataTemplateGenerator.run(System.getProperty(AbstractGenerator.GENERATOR_RESOLVER_PATH),
+    String resolverPath = System.getProperty(AbstractGenerator.GENERATOR_RESOLVER_PATH);
+    if (resolverPath != null && ArgumentFileProcessor.isArgFile(resolverPath))
```

Comment:
Nice. Thanks @evanw555 for convincing me to extract `isArgFile(String)` to a utility class.